docs/enhancement-proposals/off-ep-0007.md

```diff
@@ -0,0 +1,116 @@
+# OFF-EP 0007b — Add long-range dispersion attribute in vdW section
+
+**Status:** Draft
+
+**Authors:** Matt Thompson, John Chodera
+
+**Stakeholders:** Jeffrey Wagner, David Mobley, John Chodera
+
+**Acceptance criteria:** <Unanimity>
+
+**Created:** 2022-06-18
+
+**Discussion:** [PR #40](https://github.com/openforcefield/standards/pull/40) [PR #44](https://github.com/openforcefield/standards/pull/44)
+
+**Implementation:** [``openff-standards``](https://github.com/openforcefield/openff-standards)
+
+## Abstract
+
+This change adds an attribute to the `vdW` section of the SMIRNOFF specification to clarify how long-range vdW interactions are handled.
+
+## Motivation and Scope
+
+In version 0.3 of the `<vdW>` tag, there is no exposed option for specifying how long-range vdW interactions are to be handled.
+Existing implementations of SMIRNOFF, to our knowledge, all presume an isotropic long-range dispersion correction is used,
+but important details that determine the "correct" potential energy are omitted, and no facility to specify alternative approaches to treating
+long-range interactions is provided. This proposal makes this an explicit attribute and adds specifications for the most important variants:
+no long-range vdW correction (`none`),
+an isotropic long-range vdW correction that integrates the vdW potential against the switching function out to infinite pair separation (`isotropic`),
+and the LJPME formulation (`LJPME`).
+
+## Usage and Impact
+
+While existing SMIRNOFF implementations always use a long-range isotropic dispersion correction (i.e. [OpenFF
+Toolkit](https://github.com/openforcefield/openff-toolkit/blob/0.10.6/openff/toolkit/typing/engines/smirnoff/parameters.py#L3695)),
+following discussion in [Standards #38](https://github.com/openforcefield/standards/pull/38), it was recognized that
+(1) there was no way to disable this long-range correction or specify an important alternative approach (LJPME) that leads to a different potential definition,
+(2) the exact form of the long-range isotropic dispersion correction currently used was not specified as part of the SMIRNOFF standard.
+Updating the specification would not impact this behavior if using the default, recommended setting.
+
+This proposed enhancement would provide a way to disable the long-range vdW correction as well as build force fields that use LJPME,
+which has been recognized as being important in treating heterogeneous systems such as [those containing lipid bilayers](https://doi.org/10.1021/ct400140n).
+
+Most engines implement a common form of isotropic long-range vdW correction in which the pairwise vdW potential is averaged over all vdW site pairs
+and integrated (numerically or analytically) alongside a surface area term (and a term to account for the switching function if one is in use) to account for the missing interactions out to infinite separation:
```

Comment:
I'm not sure what you mean here.  It's just a volume integral, isn't it, with g(r) =1 in the switching region and beyond?